torch_geometric/nn/conv/cheb_conv.py

```diff
@@ -113,6 +111,13 @@ def __norm__(self, edge_index, num_nodes: Optional[int],
                                                 normalization, dtype,
                                                 num_nodes)
 
+        if lambda_max is None:
+            lambda_max = 2.0 * edge_weight
```

Comment:
Shouldn't this be?
```python
lambda_max = torch.tensor(2.0, dtype=dtype, device=edge_index.device)
```

Reply:
### Sorry, I made a mistake. It should be:
`lambda_max = 2.0 * edge_weight.max()`
If normalization is 'sym' or 'rw' then `edge_weight.max()` equals 1.